daffodil-tdml-processor/src/main/scala/org/apache/daffodil/processor/tdml/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -198,15 +200,17 @@ class DaffodilTDMLDFDLProcessor private (private var dp: DataProcessor)
     copy(dp = dp.withDebugger(d))
   }
 
-  override def withValidationMode(
-    validationMode: ValidationMode.Type
+  override def withValidator(
+    validator: api.validation.Validator
```

Comment:
Yeah, this right here is maybe a bit questionable. The TDML DFDL Processor API probably doesn't want to require a Daffodil Validator, but something else. Not sure exactly how to make that work. Just open a bug and we can figure out what the right change is.